### ISSUE TITLE
Fix compatibility with Django 1.11.2 (*** AttributeError: Can only be accessed via an instance.)

### ISSUE THREAD

**Issue by seanchon** [gracenote/django-health-monitor#2]
The `get_or_create` method is failing with the message `*** AttributeError: Can only be accessed via an instance.` in Django 1.11.2.

Steps:

- Add CI coverage in Travis CI for `env: DJANGO="Django<1.12"`.

```
    @classmethod
    def create(cls, uid, **kwargs):
        health_test = cls(uid=uid, time=timezone.now(), **kwargs)
        health_test.save()

        h, _ = cls.health_model.objects.get_or_create(uid=uid)
        h.update_score(test=cls.test, score=cls.get_score(**kwargs))

        return health_test
```

**Comment by seanchon** [comment]
Addressed in https://github.com/gracenote/django-health-monitor/commit/abafa38a4abc7b981326940780cda77728092769.

**Comment by seanchon** [comment]
Fix is in 0.2.8.
